src/VideoComponents/AvailabilityWidget.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import './css/AvailabilityWidget.css';
+
+const textStyle = {
+  color: '#FFFFFF',
+  fontWeight: 'bold',
+};
+
+const textStyleOff = {
+  color: '#A0A0A0',
+  fontWeight: 'lighter',
+};
```

Comment:
Could be an idea to add classes instead of inline styles. That makes everything related to style be in .css which makes it easy to find. No big deal tho 👍